probes/contributorsFromOrgOrCompany/impl_test.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:stylecheck
+package contributorsFromOrgOrCompany
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/clients"
+	"github.com/ossf/scorecard/v4/finding"
+)
+
+type User struct {
+	Login            string
+	Companies        []string
+	Organizations    []User
+	NumContributions int
+	ID               int64
+	IsBot            bool
+}
+
+func Test_Run(t *testing.T) {
+	t.Parallel()
+	// nolint:govet
+	tests := []struct {
+		name     string
+		raw      *checker.RawResults
+		outcomes []finding.Outcome
+		err      error
+	}{
+		{
+			name: "Should be 'comp1', 'comp2', 'Login'",
+			raw: &checker.RawResults{
+				ContributorsResults: checker.ContributorsData{
+					Users: []clients.User{
+						{
+							Companies: []string{"comp1", "comp2"},
+							Organizations: []clients.User{
+								{
+									Login:            "Login",
+									Companies:        []string{"comp3", "comp4"}, // These should not be included
```

Comment:
this might just be me missing something, but why should these be excluded?

Reply:
It may not be intentional, but this is how the check currently works: https://github.com/ossf/scorecard/blob/9c9c84b020b472d9cd7601e55c5a29cc06ac0219/checks/evaluation/contributors.go#L42-L54

It does not check for `User.Organizations.Companies`

Reply:
Ah, didn't realize that organizations are users.